emudev_resources_systems.md

```diff
@@ -379,10 +377,8 @@ There are no "full" tutorials for other systems, so using references will be a b
 - <https://github.com/redcode/Z80/wiki/Tests>
 
 ## Misc
-- Generated instruction tests for the SM83 (Gameboy) / Z80 / SPC700 (SNES & misc)
-<https://github.com/raddad772/jsmoo/tree/main/misc/tests/GeneratedTests>
-- Instruction tests in json format for 65[c]02, 65816, 68000 and SPC700:
```

Comment:
Can you add the details back about how they're instruction tests in JSON format generated from an emulator?

Reply:
OK, done, take a look.